sosw/managers/task.py

```diff
@@ -80,7 +80,8 @@ class TaskManager(Processor):
                 'closed_at':           'N',
                 'desired_launch_time': 'N',
                 'arn':                 'S',
-                'payload':             'S'
+                'payload':             'S',
+                'log_stream_name':     'S'
```

Comment:
1) This is not `log_steam_name`. This is `scheduler_log_stream_name`. The two BIG differences. This doesn't bring any real value without the `scheduler_invocation_id`... I think, this data might be interesting, but far less interesting than the list of streams and invocation_ids for each attempt. These ones could be normally saved when a new Processor call takes starts the invocation and calls the `worker_assistant` to register (I started working on this task).

This is not currently designed and implemented.